src/main/java/appeng/container/implementations/ContainerSpatialIOPort.java

```diff
@@ -85,6 +93,22 @@ public void detectAndSendChanges()
 					this.setMaxPower( (long) ( 100.0 * eg.getMaxStoredPower() ) );
 					this.setRequiredPower( (long) ( 100.0 * sc.requiredPower() ) );
 					this.setEfficency( (long) ( 100.0f * sc.currentEfficiency() ) );
+
+					final DimensionalCoord min = sc.getMin();
+					final DimensionalCoord max = sc.getMax();
+
+					if( min != null && max != null && sc.isValidRegion() )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS134)